data/pages-content/cfp.md

```diff
@@ -0,0 +1,72 @@
+
+To see the type of topics presented at previous EuroPython events, check out our [past EuroPython conferences](https://www.europython-society.org/europython/).
+Browse through our past [talk videos](https://www.youtube.com/c/EuroPythonConference) for more inspiration.
+
+For PyData related ideas, take a look at PyData events: [pydata.org](https://pydata.org/).
+
+### Open Source ###
+As a reminder, EuroPython presentations are intended to share knowledge and experience. To this end, we encourage the code and/or data that your talk relies on to be open-source. Ideally, the audience would have access to the necessary tools to reproduce the results of the talk. We welcome talks focused on the practical application of tools and concepts you use; but we discourage sales-oriented proposals whose main aim is to sell a product.
+
+## Selection Process ##
+
+We believe that a good conference consists of popular talks as well as new or niche topics - all by and for our diverse community. Thus, after Call for Proposals (CFP) finishes, submissions go through two rounds of review and refinement as we put together our programme. This is a "blind" process, so speakers' names will not be displayed to the reviewers.
+
+* **community voting**: open to all 2015-2023 EuroPython conference ticket holders + 2023 proposal submitters. Submissions are in random order for each voter. Please check out our talk voting page for details.
+
+* **panel feedback**: a diverse group of experienced reviewers will refine and give feedback on the submissions within their realm of expertise. Their insights help the programme team refine and plan a programme that truly reflects, informs and stimulates our EuroPython family. Submit your interest here
+
+Ultimately, the programme team curate the final programme based upon these important factors: community voting, panel feedback and editorial criteria such as the diversity, depth and breadth of topics and speakers.
+
+## First-Time Speakers ##
+
+We encourage submissions by first-time speakers, especially those from under-represented backgrounds. Towards the goal, we will run a **Speaker Mentorship Programme** and match mentees with experienced mentors. who can offer advice and feedback on talk ideas/presentations. The Mentorship Programme will run two workshops: one close to the CFP closing date, focusing on improving your submissions; the other close to the conference date, helping you improve your talk.
+
+Mentorship Programme sign-up starts on 22 February. More details will be announced soon!
+
+[comment]: # (The below is not decided ## Speaker Tickets ##)
+[comment]: # (We are hoping to meet as many speakers as possible in-person. To make travel easier, **accepted speakers are eligible for a free ticket to attend the conference**. Travel and accommodation expenses will not be reimbursed; but we will have a Financial Aid programme and encourage speakers, especially those from under-represented backgrounds to apply.)
+
```

Comment:
Can we add a wee section which directs people to "programme@europython.eu" should they have any questions?

Reply:
added up top.